src/java.desktop/macosx/classes/sun/lwawt/macosx/CAccessibility.java

```diff
@@ -519,6 +519,17 @@ public AccessibleAction call() throws Exception {
         }, c);
     }
 
+    // This method is called from the native in CommonComponentAccessibility.m
+    private static int getAccessibleActionCount(final AccessibleAction aa, final Component c) {
+        if (aa == null) return 0;
+
+        return invokeAndWait(new Callable<Integer>() {
+            public Integer call() throws Exception {
+                return aa.getAccessibleActionCount();
+            }
+        }, c);
```

Comment:
There's a more compact way of doing this:
```suggestion
        return invokeAndWait(aa::getAccessibleActionCount, c);
```

Reply:
Done